src/test/java/uk/gov/hmcts/reform/sscs/utils/SscsCaseDataUtilsTest.java

```diff
@@ -0,0 +1,381 @@
+        Mockito.when(sscsCaseData.getAppeal()).thenReturn(appeal);
+        //then
+        List<String> facilitiesRequired = SscsCaseDataUtils.getFacilitiesRequired(sscsCaseData);
+        assertEquals(3, facilitiesRequired.size());
+        assertEquals("signLanguageInterpreter", facilitiesRequired.stream().findFirst().orElseThrow());
+        assertEquals("disabledAccess", facilitiesRequired.stream().skip(2).findFirst().orElseThrow());
+    }
+
+    @Test
+    void shouldGetEmptyFacilitiesRequiredWhenAppealNull() {
+        // given
+        SscsCaseData sscsCaseData = Mockito.mock(SscsCaseData.class);
+        // when
+        Mockito.when(sscsCaseData.getAppeal()).thenReturn(null);
+        //then
+        assertTrue(SscsCaseDataUtils.getFacilitiesRequired(sscsCaseData).isEmpty());
+    }
+
+    @Test
+    void shouldGetCaseName() {
+        // given
+        SscsCaseData sscsCaseData = Mockito.mock(SscsCaseData.class);
+        Appeal appeal = Mockito.mock(Appeal.class);
+        Appellant appellant = Mockito.mock(Appellant.class);
+        Name name = Mockito.mock(Name.class);
+        // when
+        Mockito.when(name.getFullName()).thenReturn("Mr Harry Potter");
+        Mockito.when(appellant.getName()).thenReturn(name);
+        Mockito.when(appeal.getAppellant()).thenReturn(appellant);
+        Mockito.when(sscsCaseData.getAppeal()).thenReturn(appeal);
+        //then
+        assertEquals("Mr Harry Potter", SscsCaseDataUtils.getCaseName(sscsCaseData));
+    }
+
+    @Test
+    void shouldGetEmptyCaseNameWhenAppealNull() {
+        // given
+        SscsCaseData sscsCaseData = Mockito.mock(SscsCaseData.class);
+        // when
+        Mockito.when(sscsCaseData.getAppeal()).thenReturn(null);
+        //then
+        assertEquals("",SscsCaseDataUtils.getCaseName(sscsCaseData));
+    }
+
+    @Test
+    void shouldGetNumberOfAttendees() {
+        // given
+        SscsCaseData sscsCaseData = Mockito.mock(SscsCaseData.class);
+        Appeal appeal = Mockito.mock(Appeal.class);
+        HearingSubtype hearingSubtype = Mockito.mock(HearingSubtype.class);
+        HearingOptions hearingOptions = Mockito.mock(HearingOptions.class);
+        Representative representative = Mockito.mock(Representative.class);
+        // when
+        Mockito.when(representative.getHasRepresentative()).thenReturn(YesNo.YES.getValue());
+        Mockito.when(hearingOptions.getWantsToAttend()).thenReturn(YesNo.YES.getValue());
+        Mockito.when(hearingSubtype.isWantsHearingTypeFaceToFace()).thenReturn(true);
+        Mockito.when(appeal.getRep()).thenReturn(representative);
+        Mockito.when(appeal.getHearingOptions()).thenReturn(hearingOptions);
+        Mockito.when(appeal.getHearingSubtype()).thenReturn(hearingSubtype);
+        Mockito.when(sscsCaseData.getAppeal()).thenReturn(appeal);
+        //then
+        assertEquals(3, SscsCaseDataUtils.getNumberOfPhysicalAttendees(sscsCaseData));
+    }
+
+    @Test
+    void shouldGetParties() {
```

Comment:
I kinda see why this is so big.. it's a far reaching method and we need to set up all the mocks for it but it still freaks me out.